packages/contract-method-name/util/fetch-helper.js

```diff
@@ -0,0 +1,20 @@
+import fetch from 'node-fetch'
```

Comment:
better detect window.fetch first and use dynamic import to import node-fetch
then we can mock a empty node-fetch when use this in fluent